pkg/bloombuild/protos/types.proto

```diff
@@ -27,12 +28,18 @@ message DayTable {
   string prefix = 2;
 }
 
+message ProtoSeries {
+  uint64 fingerprint = 1;
+  repeated logproto.ShortRef chunks = 2;
+}
+
 message ProtoGapWithBlocks {
   ProtoFingerprintBounds bounds = 1 [
     (gogoproto.nullable) = false,
     (gogoproto.jsontag) = "bounds"
   ];
-  repeated string blockRef = 2;
+  repeated ProtoSeries series = 2;
```

Comment:
During a rollout, this may make builders running an older version to fail to deserialize the task. This is fine: task will be retried and eventually the builder will restart.